### DISCUSSION TITLE
List View Strategy

### DISCUSSION THREAD

**Discussion by acbee** [pymedusa/Medusa#9543] [Q&A]
Long time user, first time commenter :)

I've noticed a number of changes lately to the way list views operate or look including the schedule and show detail episodes and history. 

Is there an overall strategy behind these changes that's articulated somewhere? 

Ref: https://github.com/pymedusa/Medusa/discussions/9519

**Comment by p0psicles** [comment]
All table layouts need to be converted to vue.js. a modern js framework we have chosen to replace the old server side rendered python HTML template. And in stead of re-programming each table we have chosen to use vue-good-table as a table component.

So each page that has a table layout, is transitioned to the vue-good-table. I try to keep all old functionality as much as possible. But that's not always possible

**Reply by acbee** [reply]
Thanks! I saw some references to "vueify" and assumed it was part of moving to vue.js. I'm not familiar with vue-good-table so I'll do some more reading :)
